packages-ts/gauntlet-terra-contracts/src/commands/contracts/proxy_ocr2/inspection/inspect.ts

```diff
@@ -0,0 +1,155 @@
+import { inspection, logger } from '@chainlink/gauntlet-core/dist/utils'
+import { InspectInstruction, instructionToInspectCommand } from '../../../abstract/inspectionWrapper'
+import { RDD } from '@chainlink/gauntlet-terra'
+import { CATEGORIES } from '../../../../lib/constants'
+import { CONTRACT_LIST } from '../../../../lib/contracts'
+import { LCDClient } from '@terra-money/terra.js'
+
+type CommandInput = {
+  aggregator: string
+  version: string
+  description: string
+}
+
+type ContractExpectedInfo = {
+  aggregator: string
+  version: string
+  description: string
+  phaseId?: number
+  decimals?: string | number
+  owner?: string
+  latestRoundData?: object
+}
+
+const makeInput = async (flags: any, args: string[]): Promise<CommandInput> => {
+  if (flags.input) return flags.input as CommandInput
+  const rdd = RDD.getRDD(flags.rdd)
+  const contract = args[0]
+  const info = rdd.proxies[contract]
+
+  return {
+    aggregator: info.aggregator,
+    version: info.contractVersion,
+    description: info.name,
+  }
+}
+
+const makeInspectionData = () => async (input: CommandInput): Promise<ContractExpectedInfo> => {
+  return {
+    aggregator: input.aggregator,
+    version: input.version,
+    description: input.description,
+  }
+}
+
+const makeOnchainData = (provider: LCDClient) => async (
+  instructionsData: any[],
+  input: CommandInput,
+  proxyContract: string,
+): Promise<ContractExpectedInfo> => {
+  const aggregator = instructionsData[0]
+  const version = instructionsData[1]
+  const description = instructionsData[2]
+  const phaseId = instructionsData[3]
+  const decimals = instructionsData[4]
+  const owner = instructionsData[5]
+  const latestRoundData = instructionsData[6]
+
+  return {
+    aggregator,
+    version,
+    description,
+    phaseId,
+    decimals,
+    owner,
+    latestRoundData,
+  }
+}
+
+const inspect = (expected: ContractExpectedInfo, onchainData: ContractExpectedInfo): boolean => {
+  let inspections: inspection.Inspection[] = [
+    inspection.makeInspection(onchainData.aggregator, expected.aggregator, 'Aggregator'),
+    inspection.makeInspection(onchainData.version, expected.version, 'Version'),
+    inspection.makeInspection(onchainData.description, expected.description, 'Description'),
+  ]
+
+  logger.line()
+  logger.info('Inspection results:')
+  logger.info(`Aggregator: 
+    - Aggregator: ${onchainData.aggregator}
+  `)
+
+  logger.info(`Version: 
+    - Version: ${onchainData.version}
+  `)
+
+  logger.info(`Description: 
+    - Description: ${onchainData.description}
+  `)
+
+  logger.info(`PhaseID: 
+    - PhaseID: ${onchainData.phaseId}
+  `)
+
+  logger.info(`Decimals: 
+    - Decimals: ${onchainData.decimals}
+  `)
+
+  logger.info(`Owner: 
+    - Owner: ${onchainData.owner}
+  `)
+
+  logger.info(`Last Round Data:
+    - Data: ${JSON.stringify(onchainData.latestRoundData)}
```

Comment:
Let's print the exact details here:
```js
  if (onchainData.latestRoundData) {
    logger.info(`Last Round Data:
    - Round ID: ${onchainData.latestRoundData?.roundId}
    - Answer: ${Number(onchainData.latestRoundData?.answer) / 10 ** Number(onchainData.decimals)}
    - Transmission at: ${new Date(onchainData.latestRoundData.transmissionTimestamp * 1000)}
    - Observation at: ${new Date(onchainData.latestRoundData.observationsTimestamp * 1000)}
  `)
  } else {
    logger.warn('Latest round not found')
  }
```
Notice that the timestamp comes in Unix format. I'd recommend to have a `dateFromUnix` helper function into `/lib/utils.ts` as we've been seeing it for some time now. If you do, please add a "TODO" to move that to `gauntlet-core` at some point